lix/packages/sdk/src/change-queue.test.ts

```diff
@@ -149,46 +145,46 @@ test("should use queue and settled correctly", async () => {
 		.select("snapshot.value")
 		.execute();
 
+	const updatedEdges = await lix.db
+		.selectFrom("change_edge")
+		.selectAll()
+		.execute();
+
 	expect(updatedChanges).toEqual([
-		{
-			id: updatedChanges[0]?.id,
-			created_at: updatedChanges[0]?.created_at,
-			snapshot_id: updatedChanges[0]?.snapshot_id,
-			parent_id: null,
+		expect.objectContaining({
 			entity_id: "test",
 			type: "text",
 			file_id: "test",
 			plugin_key: "mock-plugin",
 			value: {
-				text: "test",
+				text: "insert text",
 			},
-		},
-		{
+		}),
+		expect.objectContaining({
```

Comment:
I updated the tests to `expect.objectContaining` to reduce the refactor effort in the future.